src/cross_section_measurement/make_binning_plots.py

```diff
@@ -74,10 +76,62 @@ def make_scatter_plot( input_file, histogram, channel, variable, title ):
     for output_format in output_formats:
         plt.savefig( output_folder + save_as_name + '.' + output_format )
 
+def makePurityStabilityPlots(input_path, channel, variable):
+    global output_folder, output_formats
+    
+    inputFile = '%s/binningInfo_%s_%s.txt' % ( input_path, variable, channel)
+    info = read_data_from_JSON(inputFile)
+    print input_path, channel, variable
+    print info
+    print info['s_i']
+    print bin_edges[variable]
```

Comment:
quite the few print statements (a few more below). Are they really needed?

Also, if possible use functions from ```tools.plotting```. If they are not available, create a feature request ticket so that this code can be moved into ```tools.plotting```